### PR TITLE
fix: addressing some of the digipower issues

### PR DESCRIPTION
**Pull request by streitlua**
Changes:
- added HSL experience (https://github.com/hestiaAI/digipower-data-experiences/issues/57)
- added LinkedIn experience (https://github.com/hestiaAI/digipower-data-experiences/issues/58)
- changed GDV treeview items to only expand on click of the arrow to allow copying text (https://github.com/hestiaAI/digipower-data-experiences/issues/68)
- changed GDV to display name of the file in file details (https://github.com/hestiaAI/digipower-data-experiences/issues/73)
- changed csv parser to also use `;` or `\t` as a delimiter (https://github.com/hestiaAI/digipower-data-experiences/issues/61)
- style changes (https://github.com/hestiaAI/digipower-data-experiences/issues/59):
  - changed title of digipower instance to #digipower data experiences
  - added padding to card images and removed `shaped` property
  - moved hestialabs and partner logos to the right of navigation bar
  - removed home icon from navigation bar on the home page
  - removed advanced view button from navigation drawer

### CODE REVIEW

**Comment by valentinoli** [comment]
Hey @streitlua 

a933544
I put `appName` in the config instead of having it as a global environment variable.
And the default is "HestiaLabs Experiences" if none is specified.
Set "#digipower Data Experiences" as the app name for the digipower instance.
Also did some refactoring of the meta tags definitions.

File: layouts/default.vue [hunk 1]
```diff
@@ -56,11 +56,31 @@ export default {
...
+        {
+          hid: 'twitter:card',
+          property: 'twitter:card',
+          content: 'summary'
+        },
+        {
+          hid: 'twitter:site',
+          property: 'twitter:site',
+          content: '@HestiaLabs'
+        },
```
Comment (valentinoli) [inline layouts/default.vue:78]: We might want to move these two tags to `meta.twitterCard` and `meta.twitterSite` in `nuxt.config.js`

**Comment by streitlua** [comment]
I noticed a problem with the csv parser for some of the `;`-separated files (accounttransactions.csv for instance), I'm investigating.

**Comment by valentinoli** [comment]
@streitlua 
In commit b19953c the commit message references issues in this repository but I suppose they were meant to reference the other repo. Not serious, just thought you should be aware of this. I think we need to clarify where issues are recorded. An issue should be recorded in this repo if it deals with UI changes.

**Comment by streitlua** [comment]
@valentinoli you are right, I put the issue numbers of the https://github.com/hestiaAI/digipower-data-experiences repo, and in the description of this PR I linked the correct issues.
It's a bit confusing to manage these two repos that interact together, and I don't think it would be ideal to record every issue in both.

**Pull request merged**